code/obj/item/clothing/suits.dm

```diff
@@ -1214,6 +1214,24 @@ TYPEINFO(/obj/item/clothing/suit/hazard/fire/armored)
 		..()
 		setProperty("space_movespeed", 1.5)
 
+/obj/item/emergencysuitfolded
+	name = "folded emergency suit"
+	desc = "A suit that protects against low pressure environments for a short time. At least, it would be if it hadn't been vacuum-compressed into a small rectangle. You'll have to unfold it before putting it on."
+	icon = 'icons/obj/clothing/overcoats/item_suit_hazard.dmi'
+	icon_state = "emerg_folded"
+	item_state = "emerg"
+	w_class = W_CLASS_TINY
+
+	attack_self(mob/user as mob)
+
+		user.drop_item(src) //clear hands
+		boutput(user, SPAN_NOTICE("You deploy the [src]!"))
```

Comment:
"deploy" sounds weird here, "unfold" would probably be better

Reply:
it originally did, but in that case the message would be "you unfold the folded emergency suit" which seemed a little repetitive